.github/workflows/release-plz.yml

```diff
@@ -29,3 +29,5 @@ jobs:
       - uses: ./.github/actions/cronjob-setup
       - name: Trigger tarball builds of cargo-quickinstall
         run: make recheck-self-only
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```
gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:\n  env:\n    GH_TOKEN: ${{ github.token }}
```

Reply:
interesting. I just copy-pasted that from somewhere else and it seems to work. I guess the cronjob uses GH_TOKEN so go for it